libraries.txt

```diff
@@ -8,7 +8,21 @@
 # matches, so we *don't* have the full range of requirements.txt supported
 # syntax available.
 
-matplotlib==3.5.1
-numpy==1.22.3
-pandas==1.4.2
-typing-extensions==3.10.0.0
+# List from https://github.com/srobo/robot-image/blob/main/files/python/libraries.txt
+matplotlib==3.7.2
+# Note: pandas 2.1 has some known issues, avoid it for now.
+# (https://github.com/pandas-dev/pandas/issues?q=is%3Aissue+is%3Aopen+label%3ARegression)
+pandas==2.0.3
+numpy==1.24.4
+pillow==10.0.0
+shapely==2.0.1
+scipy==1.11.2
+scikit-learn==1.3.0
```

Comment:
I notice flask is in https://github.com/srobo/robot-image/blob/main/files/python/libraries.txt but omitted here, is that intentional (I'm not sure why they'd need flask anyway, to be honest)

Reply:
I think flask is potentially useful on the robots if teams want to integrate with/provider their own web UI. I don't think it's useful in the simulator, so it's intentional to drop it. I should have added a comment about that.
https://github.com/srobo/competition-simulator/pull/400/commits/b2f61cfd9ab47da3abbc6caca85ef6ebec19739c